portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.Comment;
+import com.google.sps.data.Conversation;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comments")
+public class CommentServlet extends HttpServlet {
+
+  private Conversation converse = new Conversation();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+    String json = new Gson().toJson(converse);
+    response.getWriter().println(json);
+  }
+
+  @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Indentation - suggest trying to get the formatter up and running?

Reply:
Added indent -- I'll see if I can get it up and running